test/e2e/apps/job.go

```diff
@@ -361,6 +367,52 @@ var _ = SIGDescribe("Job", func() {
 			framework.ExpectEqual(pod.Status.Phase, v1.PodFailed)
 		}
 	})
+
```

Comment:
a comment with a link to the issue that it repros will be good
https://github.com/kubernetes/kubernetes/issues/106884